src/Shuttle.h

```diff
@@ -56,41 +58,53 @@ class CommandParameters;
 \brief Shuttle that deals with parameters.  This is a base class with lots of
```

Comment:
Update comment "Shuttle"?

Reply:
Yes I should, and also comments in ShuttleAutomation.h and EffectAutomationParameters.h and ShuttleGetDefinition.h

Reply:
Thanks